src/CurrentAndNextFeeSchedule.js

```diff
@@ -0,0 +1,62 @@
+import * as proto from "@hashgraph/proto";
+export default class CurrentAndNextFeeSchedule {
+    /**
+    * @param {object} props
+    * @param {object} props.currentFeeSchedule
```

Comment:
These should be
```javascript
    /**
    * @param {object} [props]
    * @param {FeeSchedule} [props.current]
    * @param {FeeSchedule} [props.next]
    */
    constructor(props = {}) { ... }
```
This means the fields are of  type `FeeSchedule` which will require you to do `FeeSchedule._fromProtobuf(...)` in the this class's `_fromProtobuf()` method.